src/common/scrub_types.cc

```diff
@@ -158,12 +158,22 @@ void inconsistent_snapset_wrapper::set_headless()
 
 void inconsistent_snapset_wrapper::set_ss_attr_missing()
```

Comment:
@dzafman gotcha. let's keep it as it is then, to be more consistent with the user facing API. i was tempted to use "snapset" instead of "ss" at seeing your change, feeling it could be more readable. =)